util/deployment/create-deployment-issue

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -e
+
+# Ensure we are in the project root
+if [ ! -f "GEMINI.md" ] || [ ! -d "infra" ] || [ ! -d "frontend" ] || [ ! -d "backend" ]; then
+  echo "Please run this script from the root of the webstatus.dev repository."
+  exit 1
+fi
```

Comment:
Nit: You can use the same check for checking if in root as the other scripts.

Reply:
Indeed, fixed in the most recent push. Thanks!